src/util.rs

```diff
@@ -252,30 +302,34 @@ pub(crate) fn get_last_error() -> String {
     }
 }
 
-#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Debug)]
-pub struct Version {
-    pub major: u16,
-    pub minor: u16,
-}
-
-impl std::fmt::Display for Version {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}.{}", self.major, self.minor)
-    }
+pub(crate) fn set_adapter_mtu(name: &str, mtu: usize) -> std::io::Result<()> {
+    // command line: `netsh interface ipv4 set subinterface "MyAdapter" mtu=1500 store=persistent`
+    let args = &[
+        "interface",
+        "ipv4",
+        "set",
+        "subinterface",
+        &format!("\"{}\"", name),
+        &format!("mtu={}", mtu),
+        "store=persistent",
+    ];
+    run_command("netsh", args)?;
+    Ok(())
 }
 
-/// Returns the major and minor version of the wintun driver
-pub fn get_running_driver_version(wintun: &crate::Wintun) -> Result<Version, crate::Error> {
-    let version = unsafe { wintun.WintunGetRunningDriverVersion() };
-    if version == 0 {
-        Err(crate::Error::from(get_last_error()))
-    } else {
-        let v = version.to_be_bytes();
-        Ok(Version {
-            major: u16::from_be_bytes([v[0], v[1]]),
-            minor: u16::from_be_bytes([v[2], v[3]]),
-        })
+/// Runs a command and returns an error if the command fails, just convenience for users.
+pub fn run_command(command: &str, args: &[&str]) -> std::io::Result<Vec<u8>> {
```

Comment:
I would `#[doc(hidden)]` this since this isnt a core part of our public api. Ideally we would have this be `pub(crate)`, but then the examples can't use it.